tests/Backend/Unit/Console/ImportGreenlight2Test.php

```
@@ -208,33 +263,16 @@ protected function test_command($roomAuth, ?string $prefix = null)
-            ->expectsQuestion('Do you wish to commit the import?', 'yes')
-            ->expectsOutput('Import completed');
+        // Mock database connections with fake data
+        $this->fakeDatabase($roomAuth, new Collection($users), new Collection($rooms), new Collection($sharedAccesses), new Collection($presentations));
+
+        // Mock presentation files
+        $storageMock = Storage::fake();
+        $storageMock->put('migration/presentations/fe/en/feen6movahheegheeg0ovahche8bu3mo', 'Foobar test 123');
+        $storageMock->put('migration/presentations/xi/ve/xivei7mi0cohtoecacahyaich8ohzaed', 'Foobar test 456');
+
+        // Run artisan command and validate expectation hook
+        $rollback = $this->test_helper_command($cmdOptions, $expectHook);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$rollback'. (undefined)

(UnusedLocalVariable)